### PR TITLE
Add By-city catalog creation scripts (`denovo_map.pl`)

### PR DESCRIPTION
**Pull request by avahoffman**
Create folder and files for running by-city catalogs.

Also removes the figures from the catalog tests, because I think we have made the decision to go ahead with the city level.

### CODE REVIEW

File: bash/11-city_catalogs/CD_BA_22_09_14/05a-denovo_map.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#       (assembly options) (filtering options) [-X prog:"opts" ...]
+
+module load gcc
+module load bioperl
+export PATH=$PATH:/home/ahoffm31/scratch4-mavolio2/code_AH/stacks/bin/
+export PATH=$PATH:/home/ahoffm31/scratch4-mavolio2/code_AH/stacks-2.62/
+
+echo "Creating a by-city catalog for Cynodon (CD) in Baltimore."
+
+denovo_map.pl -M 8 -T 8 -o /home/ahoffm31/scratch4-mavolio2/scratch_AH/catalog_tests/CD_BA_22_09_14 \
```
Comment (Jennycocciardi) [inline bash/11-city_catalogs/CD_BA_22_09_14/05a-denovo_map.sh:30]: I just got worried that we needed to include -n here, as well. But you're right! As you probably know, -n defaults to the -M parameter which is correct for all of our species. Phew.

**Review by Jennycocciardi — APPROVED** [review]
Looks great! I'll add the overall catalog files to a `11-metapopulation_catalogs` folder.

**Comment by avahoffman** [comment]
Sounds great!

File: bash/11-city_catalogs/CD_BA_22_09_14/05a-denovo_map.sh (hunk 1 above)
Comment (avahoffman) [reply bash/11-city_catalogs/CD_BA_22_09_14/05a-denovo_map.sh:30]: Yes I was worried about that too!

**Pull request merged**